src/server-properties.cob

```diff
@@ -154,6 +168,10 @@ PROCEDURE DIVISION USING LK-FAILURE.
     MOVE SP-MOTD TO ENTRY-VALUE
     PERFORM AppendKeyValue
 
+    MOVE "max-players" TO ENTRY-KEY
+    MOVE MAX-PLAYERS   TO ENTRY-VALUE
```

Comment:
Welcome to COBOL, where you would reasonably assume this works, but it actually doesn't.

This currently breaks when max-players=100. If you set `max-players=100` in server.properties, start the server, and look at server.properties again it will say `max-players=00`.

I suggest adding the following to WORKING-STORAGE:

```
    01 INT-TO-STR               PIC -(9)9.
```

and then adapting this code as follows:

```
    MOVE MAX-PLAYERS TO INT-TO-STR
    MOVE FUNCTION TRIM(INT-TO-STR) TO ENTRY-VALUE
```